antenna-documentation/scripts/writeIndexRootFile.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash -e
+# Copyright (c) Bosch Software Innovations GmbH 2019.
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+
+BASEDIR=$(readlink -f "$0")
+BASEDIR=$(dirname "$BASEDIR")
+
+writeIndexFile() {
+  echo '<!DOCTYPE html>
```

Comment:
```suggestion
  cat <<EOF > index.html
<!DOCTYPE html>
<html lang="en">
￼  <head>
￼      <meta charset="utf-8">
￼      <title>SW360 Antenna Documentation</title>
￼  </head>
￼  <body>
￼  <h1>SW360 Antenna Documentation</h1>
￼  <p>The following versions of our documentation are available:</p>
￼  <ol>
EOF
```

Reply:
I would like to stick with echo for consistency with the following commands